src/seeds/seeder.service.ts

```diff
@@ -0,0 +1,21 @@
+import { Injectable } from '@nestjs/common';
+import { SeederRepository } from './seeder.repository';
+import { usersData } from './data/users';
+
+@Injectable()
+export class SeederService {
+  constructor(private readonly seederRepository: SeederRepository) {}
+
+  async startSeedsUsers(): Promise<void> {
```

Comment:
it seems that the public method should be `seedData` which will execute several private methods like `seedUsersData`